src/sagemaker_sklearn_container/serving.py

```diff
@@ -91,16 +92,25 @@ def _user_module_transformer(user_module):
 
 app = None
 
+def retry_always(exception):
+    logging.info("Retrying, expcetion is: %s" % (exception))
+    return True
+
+@retry(wait_exponential_multiplier=1000, wait_exponential_max=10000, stop_max_delay=20000, retry_on_exception=retry_always)
```

Comment:
We should set a maximum number of retries in case there is something wrong with the environment as well. Other than that looks good. Please resubmit with maximum of 5 retries.

Reply:
Currently it stops after 20 seconds -> stop_max_delay=20000. Do you want to add max retry instead? Or keep both?

Reply:
So, currently it waits between retries (in seconds):
1,2,4,8,10
and that is more then 20 seconds (1+2+4+8+10), so, the 10 second wait is the last one.
It should be 5 retries together